### PR TITLE
Photonize Reps and ObjectInspector.

### PR DESCRIPTION
**Pull request by nchevobbe**
This patch changes a few CSS variables to match the Devtools Photon design spec.
We take this as an opportunity to have more consistent look and classnames across
the different reps (e.g. window, document and location).

Updated screenshot: 
screen shot 2017-09-13 at 18 46 27

### CODE REVIEW

**Comment by nchevobbe** [comment]
ni @gabrielluong to stamp it :)

**Comment by nchevobbe** [comment]
@gabrielluong this should be in better shape for review now

File: packages/devtools-reps/src/reps/reps.css [hunk 1]
```diff
@@ -6,14 +6,15 @@
...
-  --string-color: var(--theme-highlight-orange);
+  --string-color: var(--theme-highlight-red);
   --null-color: var(--theme-comment);
-  --object-color: var(--theme-body-color);
+  --object-color: var(--theme-highlight-blue);
   --caption-color: var(--theme-highlight-blue);
-  --location-color: var(--theme-content-color1);
+  --location-color: var(--theme-comment);
   --source-link-color: var(--theme-highlight-blue);
   --node-color: var(--theme-highlight-bluegrey);
```
Comment (gabrielluong) [inline packages/devtools-reps/src/reps/reps.css]: We need to find a new color for this since bluegrey is no longer a photon color

Comment (nchevobbe) [reply packages/devtools-reps/src/reps/reps.css]: ✅

**Pull request merged**